backend/src/config/index.js

```diff
@@ -2,111 +2,106 @@ import dotenv from 'dotenv'
 import links from './links.js'
 import metadata from './metadata.js'
 
+// Load env file
 if (require.resolve) {
-  // are we in a nodejs environment?
   try {
     dotenv.config({ path: require.resolve('../../.env') })
   } catch (error) {
-    if (error.code !== 'MODULE_NOT_FOUND') throw error
-    console.log('WARN: No `.env` file found in /backend') // eslint-disable-line no-console
+    if (error.code === 'MODULE_NOT_FOUND') {
```

Comment:
Finally someone caught this error. Please give a hint how to fix it: 
`cp backend/.env.template .env`